contracts/protocol/v1/oracles/P1OracleInverter.sol

```diff
@@ -58,6 +65,9 @@ contract P1OracleInverter is
         _ORACLE_ = oracle;
         _READER_ = reader;
         _ADJUSTMENT_ = adjustment;
+
+        bytes32 oracleAndAdjustment = bytes32(bytes20(oracle)) | bytes32(adjustment);
```

Comment:
must make sure the adjustment is no larger than 96 bytes. perhaps the easiest way is just making the type in the constructor `uint96`